### ISSUE TITLE
Flathub inclusion (Linux)

### ISSUE THREAD

**Issue by FestplattenSchnitzel** [pavlobu/deskreen#26]
Add Deskreen to [Flathub](https://flathub.org/home).

[Some information how to submit to Flathub.](https://github.com/flathub/flathub/wiki/App-Submission)

**Comment by pavlobu** [comment]
Yes we will slowly move to direction on making it in as many as possible package managers and open source catalogs.
Thank you for sharing instructions here.

**Comment by rugk** [comment]
Yeah, let's see whether this is possible to run in such a sandboxed environment. (But flatpaks can at least trigger screen recording, this is a thing I know, so maybe that's possible.)

**Comment by dtantono** [comment]
Hi, I hope this app also available in snap package as well

**Comment by rugk** [comment]
Naw, I personally would prefer flatpak. In contrast to snap, you can also self-host it, so you stay in control, and it is widely supported. (snap is not so nice to setup in many distros and snap's security depends on AppArmor, which is not always available in many distros)
Also – in contrast to snaps – flatpaks do not only claim to be distro-independent, but [actually are](https://kamikazow.wordpress.com/2018/06/08/adoption-of-flatpak-vs-snap-2018-edition/).
Furthermore even [Linux Mint criticizes snap](https://blog.linuxmint.com/?p=3906), because you cannot self-host a snap server as it is proprietary and (thus) also cannot modify the packages that are served centralized by Ubuntu.

**Comment by hanaral** [comment]
I'm pretty sure screen mirroring is very much possible - otherwise the Firefox, Chrome/Chromium browser and Discord flatpaks wouldn't work - let alone apps like OBS.

**Comment by G2G2G2G** [comment]
@rugk 
>Yeah, let's see whether this is possible to run in such a sandboxed environment. (But flatpaks can at least trigger screen recording, this is a thing I know, so maybe that's possible.)

flatpak has settings for the sandbox, viewable by the users. You can open the program up to everything on the system if you set a few options. Things aren't forcefully blocked.

**Comment by dtantono** [comment]
appimage is fine. But, I would love to see Steam Deck as a secondary monitor for my Ubuntu desktop with flatpak support.

**Comment by geekley** [comment]
I'm fine with snap too, but it's better to prioritize flatpak distribution on Flathub.

Even better if you can also distribute it for `apt`, since Debian/Ubuntu free/libre repositories sort of guarantee reproducibility from source code. And with something as sensitive (privacy-wise) as screen recording and transmission, reproducible builds are very important.

So, it's not just about availability or making it comfortable to update, it's also about giving users more peace of mind that the software is secure, by using sandboxing and/or (preferably) reproducible builds.
